gsms.py

```diff
@@ -205,5 +287,42 @@ def main() -> None:
                                 'custom location.')
 
 
+def get_win_path(folder_id) -> str:
```

Comment:
please add type hints...

```py
def get_win_path(folder_id: str) -> str:
```

Reply:
this was already added. most likely between you writing it and me adding it. noticed the type hint for arguments too late.